SH4RE/SH4RE/Account/EditAccountView.swift

```diff
@@ -17,27 +18,49 @@ struct EditAccountView: View {
     @State private var pfpChanged: Bool = false
     @State private var name: String = ""
     @State private var password: String = ""
-    @State private var confirmPassword: String = ""
+    @State private var newPassword: String = ""
     @State private var errorInField: Bool = false
     @State private var showPosted: Bool = false
     @State private var dataToChange:Dictionary<String, Any> = [String: Any]()
     
     func update() {
+        
+        let docRef = Firestore.firestore().collection("User Info").document(getCurrentUserUid())
+        
         dataToChange.removeAll()
         if (name != user.name && name != "") {
-            dataToChange["name"] = name
+            documentUpdate(collectionPath: "User Info", documentID: getCurrentUserUid(), data: ["name": name])
         }
-        if (pfpChanged) {
-            dataToChange["pfp"] = profilePicture
+        else if (pfpChanged) {
```

Comment:
this and below should if not else if, cause user might want to change multiple things